tools/main.cpp

```diff
@@ -279,10 +279,11 @@ int main(int argc, char **argv)
     }
 
     if ((outformat != "svg") && (outformat != "mei") && (outformat != "mei-basic") && (outformat != "mei-pb")
```

Comment:
This is getting slowly out of control. Maybe store possible values in an array or set and check against that?

Reply:
Much cleaner now, thanks!